src/Core/tests/DeviceTests/Handlers/CoreHandlerTestBaseOfT.cs

```diff
@@ -7,7 +7,8 @@ public abstract partial class CoreHandlerTestBase<THandler, TStub> : HandlerTest
 		where THandler : class, IViewHandler, new()
 		where TStub : StubBase, IView, new()
 	{
-		protected override MauiAppBuilder ConfigureBuilder(MauiAppBuilder mauiAppBuilder) =>
-			mauiAppBuilder.ConfigureTestBuilder();
+		protected override void ConfigureBuilder(MauiAppBuilder mauiAppBuilder) =>
+			mauiAppBuilder.ConfigureMauiHandlers(handlers =>
+				handlers.AddHandler<TStub, THandler>());
```

Comment:
This PR avoids the verbose code to add a handler to the mapper by making use of this new method that was added to clean up tests. Instead of using the activator randomly, we just always add the test's handler to the mapper.

This can be overridden by manually calling `EnsureHandlerCreated` and passing in the desired action.